src/main/java/org/ndgf/endit/FlushTask.java

```diff
@@ -83,14 +112,20 @@ public FlushTask(FlushRequest request, Path outDir, String type, String name)
            // <storename> and <groupname> : The store and group name of the file as provided by the arguments to this executable.  
            // <bfid>: The unique identifier needed to restore or remove the file if necessary.   
            LOGGER.debug("Send back uri: " + uri.toString());
-           return Collections.singleton(uri);
+   	   Files.deleteIfExists(requestFile);
+           
+	   return Collections.singleton(uri);
         }
         return null;
     }
 
     @Override
     public boolean abort() throws IOException
     {
-        return Files.deleteIfExists(outFile);
+        if (Files.deleteIfExists(outFile)) {
```

Comment:
return Files.deleteIfExists(outFile) && Files.deleteIfExists(requestFile);

Reply:
The same here.
If there is no objection from others (ndgf people), then I can make this change everywhere.

Reply:
so no objection from others (ndgf people) ;)

Reply:
modifications are done. Looks better :)